lib/lurker/templates/layouts/print.html.erb

```diff
@@ -20,6 +20,8 @@
   <meta name="pdfkit-footer-left" content="[title]"/>
   <meta name="pdfkit-footer-center" content="[page] of [toPage]"/>
   <meta name="pdfkit-footer-right" content="[section]"/>
+
+  <link href="<%= @service_presenter.asset_path('application.css') %>" media="all" rel="stylesheet">
```

Comment:
In HTML5 these "self-closing" slashes are optional and we actually don't use them in other places (see https://github.com/razum2um/lurker/blob/master/lib/lurker/templates/layouts/application.html.erb#L7). I removed them in meta-tags above for consistency.